sort_r.h

```diff
@@ -28,7 +28,7 @@ void sort_r(void *base, size_t nel, size_t width,
      defined __FreeBSD__ || defined __DragonFly__)
 #  define _SORT_R_BSD
 #elif (defined _GNU_SOURCE || defined __gnu_hurd__ || defined __GNU__ || \
-       defined __linux__ || defined __MINGW32__ || defined __GLIBC__)
+       defined __linux__ || defined __GLIBC__)
 #  define _SORT_R_LINUX
 #elif (defined _WIN32 || defined _WIN64 || defined __WINDOWS__)
```

Comment:
Does this line need to be updated to:

    #elif (defined _WIN32 || defined _WIN64 || defined __WINDOWS__ || __MINGW32__)

Or is the `__MINGW32__` test redundant?